.github/workflows/release.yaml

```diff
@@ -122,3 +122,38 @@ jobs:
       is_latest: ${{ needs.release-settings.outputs.is_latest == 'true' }}
       tag: ${{ github.event.release.tag_name }}
       sign: true
+      
+  release-body:
+    needs: [release-settings, publish-docker]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v4
+          
+      - name: Extract LIBS and DRIVER versions
+        run: |
+          cp .github/release_template.md release-body.md
+          LIBS_VERS=$(cat cmake/modules/falcosecurity-libs.cmake | grep 'set(FALCOSECURITY_LIBS_VERSION' | tail -n1 | grep -o '[[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*')
+          DRIVER_VERS=$(cat cmake/modules/driver.cmake | grep 'set(DRIVER_VERSION' | tail -n1 | grep -o '[[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*')
+          sed -i s/LIBSVER/$LIBS_VERS/g release-body.md
+          sed -i s/DRIVERVER/$DRIVER_VERS/g release-body.md
+          
+      - name: Append release matrixes
+        run: |
+          sed -i s/FALCOBUCKET/${{ needs.release-settings.outputs.bucket_suffix }}/g release-body.md
+          sed -i s/FALCOVER/${{ github.event.release.tag_name }}/g release-body.md
+          
+      - name: Generate release notes
+        uses: leodido/rn2md@master
+        with:
+          milestone: ${{ github.event.release.tag_name }}
+          output: ./notes.md
+        
+      - name: Merge release notes to pre existent body
+        run: cat notes.md >> release-body.md
+        
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          body_path: ./release-body.md
+          tag_name: ${{ github.event.release.tag_name }}  
```

Comment:
Remember that this entire workflow reacts to the _creation_ of a new release. Wouldn't this create another one?

Reply:
I think it will just update the existing one; perhaps i need to add `name:` input though! Thanks for the suggestion btw

Reply:
But yep i totally forgot about prereleases and draft; this should only run on final releases.
I will update the workflow.

Reply:
Ok i should've fixed it. PTAL :)

Reply:
My question is more like: is https://github.com/softprops/action-gh-release able to modify an existing release? So, have you tried creating an empty repository with a pipeline which reacts to this

```
on:
  release:
    types: [published]
```

and check if it is able to update the release body that you added? It is possible that you did or that the manual states it somewhere but from a quick look I didn't find it

Reply:
So, this is the same action used by libs; on libs, you create a release (not a tag) BUT the workflow triggers on tag. Anyway, the action updates the existing github release.
Also, the action is able to append body to an existing release (this is what we do in libs actually):  

> append_body | Boolean | Append to existing body instead of overwriting it

Reply:
So, yes the action CAN update an existing github release.

Reply:
> 💡 When the release info keys (such as name, body, draft, prerelease, etc.) are not explicitly set and there is already an existing release for the tag, the release will retain its original info.

Reply:
(added correct permissions to the job btw)